server.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+
+require('dotenv').config();
+
+const express = require('express');
+
+
+const weather = require('./data/weather.json')
+const cors = require('cors');
+
+const app = express();
+
+
+
+app.use(cors());
+
+const PORT = process.env.PORT || 3001
+
+app.get('/hello', (request, response) => { response.send('Hello, it works!') })
+
+app.get('/weather', handleWeather);
+
+app.get('/*' , (req,res) => res.status(403).send('not found'))
+
+function handleWeather(req,res) {
+  res.status(200).send(weather)
+}
+
```

Comment:
I like how you have made a function for grabbing weatherData 👍  I know you are still workin'/have a bit more work to do here to complete code for task card 2 backend
-  You can make use of  `req.query.searchQuery` here (see task card 2).
- I would also check out Express docs on `req` and `res`